test/integration/scheduler/queue_test.go

```diff
@@ -178,88 +178,93 @@ func TestSchedulingGates(t *testing.T) {
 // TestCoreResourceEnqueue verify Pods failed by in-tree default plugins can be
 // moved properly upon their registered events.
 func TestCoreResourceEnqueue(t *testing.T) {
-	// Use zero backoff seconds to bypass backoffQ.
-	// It's intended to not start the scheduler's queue, and hence to
-	// not start any flushing logic. We will pop and schedule the Pods manually later.
-	testCtx := testutils.InitTestSchedulerWithOptions(
-		t,
-		testutils.InitTestAPIServer(t, "core-res-enqueue", nil),
-		0,
-		scheduler.WithPodInitialBackoffSeconds(0),
-		scheduler.WithPodMaxBackoffSeconds(0),
-	)
-	testutils.SyncSchedulerInformerFactory(testCtx)
+	for _, qHintEnabled := range []bool{true, false} {
+		t.Run(fmt.Sprintf("TestCoreResourceEnqueue, with queuehint(%v)", qHintEnabled), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(fmt.Sprintf("queuehint(%v)", qHintEnabled), func(t *testing.T) {
```

The name of the parent test is already added by `t.Run`